### ISSUE TITLE
Foreign key references.

### ISSUE THREAD

**Issue by DeTommie** [HeidiSQL/HeidiSQL#1825]
**Is your feature request related to a problem? Please describe.**
I can't see which tabels refer to the current tabel with a foreign key.

**Describe the solution you'd like**
A new tab with a list of refferals to the current table.

**Describe alternatives you've considered**
I've used the following query to get what I need:

```
SELECT 
  TABLE_NAME,COLUMN_NAME,CONSTRAINT_NAME, REFERENCED_TABLE_NAME,REFERENCED_COLUMN_NAME
FROM
  INFORMATION_SCHEMA.KEY_COLUMN_USAGE
WHERE
  REFERENCED_TABLE_SCHEMA = '<database>' AND
  REFERENCED_TABLE_NAME = '<table>';
```

But a push of a button would be easier.

**Additional context**

**Comment by ansgarbecker** [comment]
I also think that could be useful. Only adding another tab makes the table editor overcrowded, or? Your note about the "push of a button" made me think this could be a button in the toolbar within the "Foreign keys" tab, below the "Clear" button. That could show a simple dialog with the tables referencing to the current table.

**Comment by DeTommie** [comment]
You could also do a horizontal split of the "Foreign Keys" tab, with the foreign keys on top and references in the bottom.

**Comment by AntonioCS** [comment]
I agree with @DeTommie, having the information always present would be much better.

**Comment by ansgarbecker** [comment]
I just ran the above query on my local mariadb instance, it took 21 seconds the first time it ran - that is really slow, and it may be even slower on larger instances. Second and later queries are quicker, but I wonder if there's a better performing SQL query.

**Comment by ansgarbecker** [comment]
Testing on a totally different MySQL 8.0 server now, the same query is really quick. Will test on a slower machine as well.

```sql
SELECT DISTINCT
  k.TABLE_SCHEMA, k.TABLE_NAME
  -- , TABLE_NAME,COLUMN_NAME,CONSTRAINT_NAME, REFERENCED_TABLE_NAME,REFERENCED_COLUMN_NAME -- probably better leave away
FROM
  INFORMATION_SCHEMA.KEY_COLUMN_USAGE k
WHERE
  REFERENCED_TABLE_SCHEMA = '<database>' AND
  REFERENCED_TABLE_NAME = '<table>';
```

**Comment by ansgarbecker** [comment]
Took again ~30s on a local MariaDB with some larger test databases. Should only be retrieved when the tab is visible.

I think that new list should go to the right of the existing "Foreign keys" tab. As long as the columns have a reasonable width, there is enough space on a fullhd screen:

<img width="1599" height="180" alt="Image" src="https://github.com/user-attachments/assets/cb34d977-098a-4f90-9946-1201cd4ed137" />

Note: that "referrals" listing will be a simple listing, maybe supports doubleclick on a table so you get there. But no other details besides the db.table name listed there.

**Comment by ansgarbecker** [comment]
Next builds provide a listing on the right side, as shown in the following screen, plus a button to enable and disable it:

<img width="850" height="171" alt="Image" src="https://github.com/user-attachments/assets/bb5ca8f2-9761-4936-b6ff-f7e2414e5ff5" />

The listing is currently only populated on MySQL and MariaDB. I had no luck when testing the same IS query on MS SQL.
